pyrene/src/components/Badge/Badge.tsx

```diff
@@ -33,11 +33,11 @@ const Badge: React.FC<BadgeProps> = ({
   label, maxWidth, onClick = () => null, type,
 }: BadgeProps) => (
   <div
-    styleName={clsx('badge', { [`type-${type}`]: true })}
+    className={clsx(styles.badge, { [styles[`type-${type}`]]: true })}
```

Comment:
Please do the following when the `class` will be added anyway, means always `true`.

```
className={clsx(styles.badge, styles[`type-${type}`] )}
```